style.css

```diff
@@ -1079,3 +1078,154 @@ header {
   left: -35px;
   top: 90px;
 }
+
+.contact-img2 {
+  position: relative;
+  width: 231px;
+  height: 708px;
+  left: 176px;
+  top: -727px;
+}
+
+.contact-img {
+  position: relative;
+  width: 396px;
+  height: 781px;
+  left: 11px;
+  top: 11px;
+}
+
+.contact p {
+  position: relative;
+  width: 327px;
+  height: 102px;
+  left: 48px;
+  top: -777px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact h2 {
+  position: relative;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 0;
+  margin-top: -582px;
+  top: -802px;
+  left: 89px;
+  color: #fff;
+}
+
+.contact {
```

Comment:
- [ ] Please center your container in the body element as it seems like this in 375px width screens:
![image](https://user-images.githubusercontent.com/91486791/220363292-1b2f434e-5b29-404e-b2d6-2814c46a33f0.png)
![image](https://user-images.githubusercontent.com/91486791/220363416-a85a9c09-efb8-441e-83fd-2d4552e52fbb.png)
- [ ] [**Optional**]: I also noticed your other sections besides the headline section are the same. I suggest you fix them to make it look more professional 🦾